pipeline-runner/R/gem2s-3-run_emptydrops.R

```diff
@@ -1,28 +1,52 @@
-################################################
```

Comment:
I liked this :(

Reply:
They cause issues with folding in Rstudio. If you type `alt+o` it folds by function. This breaks that.